recipes/libdai/recipe/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "libdai" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "9ba07a11af50bb4d53bceade6295a0366ef90e92440690ac52935ec144c05c70" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: libdai-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/libdai/archive/{{ version }}.tar.gz
```

Comment:
So it appears this a fork and the original source is [here]( https://bitbucket.org/jorism/libdai ). Though it appears to have been unmaintained for 1.5yrs now. The last release being nearly 3yrs ago. Seems reasonable to fork and update IMHO. Thoughts from others?

cc @conda-forge/core @conda-forge/staged-recipes

Reply:
That seems reasonable.